ssh2

```diff
@@ -36,7 +49,15 @@ if options.bust_cache or not os.path.exists(cache_file_list):
 	output = subprocess.Popen("aws ec2 describe-instances", shell=True,
 		stdout=subprocess.PIPE).stdout.read()
 	with open(cache_file_list, 'w') as f:
-		f.write(output)
+            f.write(output)
+
+if options.ident or not os.path.exists(cache_file_list):
```

Comment:
Thanks for this PR. Can you just check to see if the file exists below on line 105?